tests/ssl-opt.sh

```diff
@@ -7027,7 +7027,7 @@ run_test    "SSL async private: sign, error in resume then fall back to transpar
 
 requires_config_enabled MBEDTLS_SSL_ASYNC_PRIVATE
 requires_config_enabled MBEDTLS_SSL_RENEGOTIATION
-run_test    "SSL async private: renegotiation: client-initiated; sign" \
+run_test    "SSL async private: renegotiation: client-initiated, sign" \
```

Comment:
If a new test is added and it uses a `;`, what happens? How are developers going to know not to put `;` in their test names? Do we need something in check-names or is the failure going to be obvious enough?

Reply:
I'm planning to add a `check_tests.py` that checks test case descriptions for `;` and for uniqueness.

Reply:
I added `tests/scripts/check-test-cases.py`. There are no more `;` in test descriptions in mbedtls. There are a few in the crypto submodule which this PR does not instrument, they'll need to be fixed when we port `host_tests.function`.